test/js/pageobjects/CrossDeviceCheckYourMobile.js

```diff
@@ -0,0 +1,65 @@
+import Base from './BasePage.js'
+import {locale, verifyElementCopy} from '../utils/mochaw'
+
+class CrossDeviceCheckYourMobile extends Base {
+
+  get crossDeviceCheckYourMobileTitle() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
+  get crossDeviceCheckYourMobileSubmessage() { return this.$('.onfido-sdk-ui-crossDevice-MobileNotificationSent-submessage')}
+  get crossDeviceCheckYourMobileMayTakeFewMinutesMessage() { return this.$('.onfido-sdk-ui-crossDevice-MobileNotificationSent-boldMessage')}
```

Comment:
These functions name are so long that they become unreadable. Can we shorten them up?

Reply:
Will try :)

Reply:
Done.